Aztec/Classes/TextKit/TextView.swift

```diff
@@ -148,6 +148,17 @@ open class TextView: UITextView {
     let defaultFont: UIFont
     var defaultMissingImage: UIImage
 
+    // MARK: - Properties: Processors
+
+    /// These processors will be executed on any HTML you provide to the method `setHTML()` and
+    /// before Aztec attempts to parse it.
+    ///
+    public var preProcessors: PipelineProcessor?
+
+    /// These processors will be executed right before returning the HTML in `getHTML()`.
+    ///
+    public var postProcessors: PipelineProcessor?
```

Comment:
Nipticky: `preProcessors` and `postProcessors` both imply... multiple instances, yet, they're not arrays, but `PipelineProcessor` instances. How about if:

A. Make the `PipelineProcessor` a private struct (or nuke it entirely?), and convert the two ivars into `[Processor]` arrays, OR
B. Rename both ivars to `inputPipelineProcessor` and `outputPipelineProcessor` || `preProcessor` and `postProcessor`?